__fixtures__/test-project-rsa/web/src/Document.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 
-import { Css, Meta } from '@redwoodjs/web/dist/components/htmlTags'
-import type { TagDescriptor } from '@redwoodjs/web/dist/components/htmlTags'
+import { Css, Meta } from '@redwoodjs/web/htmlTags'
+
```

Comment:
I tried having a blank line between the regular import and the type import in a project I already had set up (using the old dist imports). I get a `There should be no empty line within import group` error. Is this no longer the case with the exports field?

Personally I prefer them grouped together (i.e. without the blank line). But we should go with whatever the linter is happy with 🧹

Reply:
I think this was find and replace doing it's thing. I'll remove the extra line!